lib/text/ttml_text_parser.js

```diff
@@ -202,38 +235,82 @@ shaka.text.TtmlTextParser = class {
    * @param {!Array.<!Element>} regionElements
    * @param {!Array.<!shaka.text.CueRegion>} cueRegions
    * @param {boolean} whitespaceTrim
+   * @param {boolean} isChild
```

Comment:
why does it matter if an element is a child?

Reply:
I should rename it `isNested` or `isNestedCue`, but it means we parse a DOM element that is child of "real" cue (with start and end attributes): like a `<span />` in a `<p start="..." end="... "/>`
It allows to bypass some checks on the cue (start and end attributes mostly since the parent DOM element have them)